package.json

```diff
@@ -7,18 +7,20 @@
     "lib": "lib"
   },
   "dependencies": {
-    "durable-functions": "^1.1.2",
-    "moment": "^2.23.0",
+    "durable-functions": "^1.1.3",
+    "moment": "^2.24.0",
     "request": "^2.88.0",
     "request-promise-native": "^1.0.5"
   },
   "devDependencies": {
-    "eslint": "^5.12.0",
-    "eslint-config-nhsuk": "^0.16.0"
+    "chai": "^4.2.0",
+    "eslint": "^5.12.1",
+    "eslint-config-nhsuk": "^0.16.0",
+    "mocha": "^5.2.0"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "eslint"
+    "test": "NODE_ENV=test mocha --exit --recursive test",
```

Comment:
I can't see where `NODE_ENV` is used. If it isn't could it be removed?

Reply:
Good spot, copy and paste error.